### PR TITLE
ADD: almalinux8 support

### PR DESCRIPTION
**Pull request by andreaspeters**
With this PR I would like to add support for building almalinux8 RPM packages. 

To start a build, we have to disable python and werror:

```bash
./build_mesos --rename --configure-flags "--disable-python --disable-werror "
```

You will also see, that I add the current path (pwd) into the create_lib_symlinks function. It never could find libmesos.so to determine lib or lib64. But I can also create a seperate PR for that if u like.

### CODE REVIEW

**Comment by ridv** [comment]
Thanks for the PR!

Re: the symlink.Let's separate it into two just because I want to make sure that it doesn't break the symlink creation on any other version. That way if it does, we can narrow it down to the commit that changes that.

Re: Alma, since Alma guarantees compatibility with RHEL, I wonder if we shouldn't just create support for RHEL 8 and make Alma a synonym. It might keep the clutter down. I'm fine with it either way though.

**Comment by andreaspeters** [comment]
Hi @ridv , thanks for feedback. 

1. I will open a second PR after this one with the symlink. :-) I test is at least with alma and centos7, but of course, it's better to be sure it would not break on other systems.
2. I really think it's a good idea to add RHEL8, even if I cannot test it. :-)  But I don't understand what u mean with "synonym". Can you explain please (sorry for that)?

**Comment by ridv** [comment]
No worries, I couldn't think of the word. I meant alias. So if you call the build script with alma8, under the hood it would just call the build for RHEL 8. That way even Rocky Linux users could take advantage of these contributions.

In essence, if it works for Alma8, it should always work for RHEL 8 (and vice versa, and anything else binary compatible with RHEL) as per my understanding, so no worries about not being able to test it 😄

**Comment by andreaspeters** [comment]
Oh ok I got it thanks. :-)  How about if I change the function "os_release" (line 191) like this:

```bash
function os_release {
  msg "Trying /etc/redhat-release..."
  if [[ -f /etc/redhat-release ]]    
  then
    # Seems to be formatted as: <distro> release <version> (<remark>)
    #                           CentOS release 6.3 (Final)
    if [[ $(cat /etc/redhat-release) =~ \
          ^(.+)' '+release' '+([^ ]+)' '+'('[^')']+')'$ ]]
    then
      local os
      case "${BASH_REMATCH[1]}" in
        'Red Hat'*) os=RedHat ;;
        'AlmaLinux'*) os=RedHat ;;
        *)           os="${BASH_REMATCH[1]}" ;;
      esac
      display_version "$os" "${BASH_REMATCH[2]}"
      return 0
    else
      err "/etc/redhat-release not like: <distro> release <version> (<remark>)"
    fi
  fi
  msg "Trying /etc/os-release..."
  if [[ -f /etc/os-release ]]
  then
.
.
.
```

Then AlmaLinux is just a RedHat and we only have to add a filter like "RedHat/8" I think. What do u think?

**Comment by ridv** [comment]
That works too, thanks!

**Comment by andreaspeters** [comment]
So, I change it. But I'm still testing so please, wait with the merge. :-)

**Comment by andreaspeters** [comment]
I'm finish with my tests. It works. :-)

File: build_mesos [hunk 1]
```diff
@@ -214,6 +209,12 @@ function os_release {
       err "/etc/redhat-release not like: <distro> release <version> (<remark>)"
     fi
   fi
+  msg "Trying /etc/os-release..."
```
Comment (ridv) [inline build_mesos:212]: Just a quick question, any reason to move this from the top to here? (Reason I'm asking is I'd rather avoid unnecessary changes so if someone looks through the history they don't get confused 😄)

**Review by ridv — COMMENTED** [review]
LGTM minus one question. Such a smaller diff, great work!

File: build_mesos (hunk 1 above)
Comment (andreaspeters) [reply build_mesos:212]: :-) Yes! On almalinux there is also a os-release file. So, if it was found, it exit the os_release release function without to check the redhat-release file.
